### ISSUE TITLE
Add option for overlay to be applied every time you load a world

### ISSUE THREAD

**Issue by NickHam13** [fragger57/creator_overlays#7]

**Comment by fragger57** [comment]
This request will be implemented in the next update as display.mode in the config file. There will be 3 display modes, 0 will be always off, 1 will be on every time you join a world, and 2 will be on every time you join a Replay Mod recording. Also just to be clear when it's set to 1 it will still be on when you join a Replay Mod recording, as it is considered a world, but I just added the third options to only display when you join a replay recording, as I imagine that most people don't what it on every time they join a world, but would still like to have it on every time they join a replay recording.
Currently this only affects the Gridlines overlay as I've sort of deemed it as the default overlay. I _may_ implement the ability to set the default overly in the config file, cause I know not everyone makes the same type of content. Some people may primarily use the Gridlines overlay, while others may primarily use the Vertical Video overlay. I'm currently working on a way to set configs in the GUI so users don't have to go into the config file every time they want to change one of the default values, so this would add some complexity to that. What I may end up doing is have the last used overlay saved in the config file and just use that as the default overlay every time you join a world or replay recording, so the user doesn't have to change that value manually. It'll just automatically update every time you change the overlay, or probably just whenever you leave a world/recording.

TLDR:
This will be implemented in the next update with 3 display modes, 0 - always off, 1 - on when joining a world/replay recording, 2 - on only when joining a Replay Mod recording. The Gridlines overlay is currently the default, and I'm considering different ways to change the default overlay.
I'm hoping to get the next update out some time this month, but no promises.
